### ISSUE TITLE
Hyperparameters with fairseq BART

### ISSUE THREAD

**Issue by shakeley** [cylnlp/dialogsum#10]
Hi, 

Thanks for the great dataset! I am reproducing the result in the paper using **fairseq**. Notice that "5,000 training steps/200 warmup steps and learning rate is set to 3e−5." 

Besides these params, I use the following params: 

> MAX_TOKENS=2048
UPDATE_FREQ=4
MAX_EPOCH=6

My results are strange (only 37 for R-1). Could you provide more details for fairseq training?

**Comment by cylnlp** [comment]
Hi @shakeley, have you seen the discussion in this [issue](https://github.com/cylnlp/dialogsum/issues/9)? And did it solve your problem?
If not, can you send your model output to [me](mailto:https://aclanthology.org/2021.findings-acl.449) so I can better see what I can do.
